src/Faithlife.Utility/StringUtility.cs

```diff
@@ -78,6 +78,38 @@ public static string ReplaceOrdinal(this string source, string oldValue, string
 #endif
 		}
 
+#if NETSTANDARD2_0
+		/// <summary>
+		/// Calls string.Contains with StringComparison.Ordinal.
+		/// </summary>
+#else
+		/// <summary>
+		/// Calls <see cref="string.Contains(char, StringComparison)">string.Contains</see> with <see cref="StringComparison.Ordinal">StringComparison.Ordinal</see>.
+		/// </summary>
+#endif
+		public static bool ContainsOrdinal(this string source, char value) =>
```

Comment:
This is exactly the same as `string.Contains(char)`: https://docs.microsoft.com/en-us/dotnet/api/system.string.contains?view=net-5.0#System_String_Contains_System_Char_

We shouldn't needlessly duplicate framework methods. It'd be OK to add a `Contains(this string, char value)` extension method for downlevel frameworks, but we should normalise the understanding that `Contains(char)` and `IndexOf(char)` _are_ ordinal comparisons.